addons/mrp/report/mrp_bom_structure_report.py

```diff
@@ -7,31 +7,42 @@
 class BomStructureReport(models.AbstractModel):
     _name = 'report.mrp.mrp_bom_structure_report'
 
+    @staticmethod
+    def _get_child_vals(record, level, qty, uom):
```

Comment:
What is `record` (which model)? A better argument name or a docstring in the method is desirable here for proper self-documentation

Reply:
@pedrobaeza updated